client/src/rr/dnssec/keypair.rs

```diff
@@ -241,11 +242,12 @@ impl KeyPair {
     /// # Return
     ///
     /// the KEY record data
+    #[allow(deprecated)]
     pub fn to_sig0key(&self, algorithm: Algorithm) -> DnsSecResult<KEY> {
         self.to_public_bytes()
             .map(|bytes| {
                      KEY::new(Default::default(),
-                              Default::default(),
+                              KeyUsage::Zone,
```

Comment:
I don't think hard-coding KeyUsage::Zone is the right solution, as Bind may use different values for KeyUsage.
In my client I'm not using to_sig0key at all, but construct the KEY using:
```
    let key_rdata = KEY::new(KeyTrust::AuthOrPrivate,
                                  KeyUsage::Entity,
                                  Default::default(), // signatory
                                  Protocol::DNSSec,
                                  Algorithm::RSASHA256,
                                  keypair.to_public_bytes().unwrap());
```

As you can see, I need KeyUsage::Entitiy.